bookshelf/tests_bookshelf.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from bookshelf_object import Bookshelf
+
+
+class BookTestCase(unittest.TestCase):
+    def setUp(self) -> None:
```

Comment:
Here you used setup for object testcase. I mean, setup for the object instance of the class `BookTestCase`.
Can you try to run entire test suite written by you (this file) but with one little modification? Just add one line in the end of this setup:

```ptyhon
def setUp(self) -> None:
        self.myBookshelf = Bookshelf("Favourite books", "Jan Kowalski", 10)
        self.myBookshelf.add_book("author2", "book title")
        print('setup')
```

How many times string `setup` appears?
What is the reason?
Try to add a class setup instead of the object setup:
```python
    @classmethod
    def setUpClass(cls) -> None:
        cls.myBookshelf = Bookshelf("Favourite books", "Jan Kowalski", 10)
        cls.myBookshelf.add_book("author2", "book title")
        print('class setup')
```

How many times string `class setup` appears?
Why?
Which one is better and why?